thundra/utils.py

```diff
@@ -241,18 +241,10 @@ def parse_x_ray_trace_info():
 def get_nearest_collector():
     region = get_configuration(constants.AWS_REGION, default="us-west-2")
```

Comment:
I think, we should not use default value here. In local env. region might be empty and in this case it is better to use `collector.thundra.io` instead of `us-west-2.collector.thundra.io`. Because at DNS level, `collector.thundra.io` returns nearest endpoint based on the current location. For ex. on my local, using `eu-central-1` (Frankfurt) region is better than using `us-west-2.collector.thundra.io` by default